Lib/test/test_tools/test_docs_version_next.py

```diff
@@ -0,0 +1,99 @@
+"""Sanity-check tests for the "docs/replace_versoin_next" tool."""
```

Comment:
```suggestion
"""Sanity-check tests for the "docs/replace_version_next.py" tool."""
```

Reply:
I'll keep this thread open until we [decide where the script will be](https://github.com/python/cpython/issues/121277#issuecomment-2242446047).